test/run_tests.py

```diff
@@ -96,68 +112,110 @@ def http_server():
         serverlog.close()
 
 
-def execute_tests(interpreter, browser, cli_options):
+def execute_tests(interpreter, browser, suite, sauceusername, saucekey):
     runner = [interpreter, '-m', 'robot.run']
     options = [opt.format(browser=browser,
                           py_version=interpreter + sys.version[:3])
                for opt in ROBOT_OPTIONS]
-    if env.TRAVIS:
-        options.extend(get_travis_conf(browser))
-    command = runner + options + cli_options + [env.ACCEPTANCE_TEST_DIR]
-    log_start(command)
+    if sauceusername and saucekey:
+        options.extend(get_sauce_conf(browser, sauceusername, saucekey))
+    command = runner
```

Comment:
This looks strange. Could either just rename `runner` to `command` originally or use something like

```python
command = runner + (['--suite', suite] if suite else [])
```